features/support/pages/shadow_root.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ShadowRoot < SitePrism::Page
+  set_url '/shadow_root.htm'
+  set_url_matcher(/shadow_root\.htm$/)
+
+  # shadow-root
```

Comment:
Remove this comment

Reply:
I fixed, https://github.com/site-prism/site_prism/pull/159/commits/175d18635c72250a82654680ec116dfa49a16b62